### ISSUE TITLE
[FR] Include shorcuts key combination meaning in scopes and navigation thumbnail toltips.

### ISSUE THREAD

**Issue by difrkaguilar** [darktable-org/darktable#13482]
**Is your feature request related to a problem? Please describe.**

The information provided by the tooltips is generally good, concise and accurate. In the particular cases of the navigation thumbnail and the scopes the options ctrl+shift+n or h as the case may be are shown. But the tooltip does not show the information for this key combination.

**Describe the solution you'd like**

It would be good, besides the information shown in the tooltips about the different combinations to use in the scopes and navigation thumbnail, to add also the information related to these specific combinations. 
For example: 

**shift+ctrl+h  hide | show scopes**
**shift+ctrl+n  hide | show navigation thumbnail** 

Although this information is also present when the h is selected and the different shortcuts are shown.

**Additional context**

![Screenshot from 2023-01-30 12-21-49](https://user-images.githubusercontent.com/67672868/215569921-3d9007ef-393d-49c7-a19b-a1bcfedc8a26.png)

![Screenshot from 2023-01-30 13-05-31](https://user-images.githubusercontent.com/67672868/215569939-9d0e5582-6575-41ee-8400-8a3a25a1e9be.png)

**Comment by Mark-64** [comment]
Yeah, the scope tooltip is quite messy, but it is most likely going to change, still working on it

**Comment by dterrahe** [comment]
The shortcut tooltips (as opposed to the mouse action tooltips) are automatically generated depending on whether a shortcut is actually set or not. It is shown more to let you know there _is_ one, than what it _does_, because the latter is often somewhat obvious. If the shortcut is for the default element and effect, those details are not mentioned for brevity and to be less distracting. My philosophy is that you can only master a program by playing around with it, so you shouldn't be afraid to try out some shortcuts; usually the effect is obvious, can be reversed by pressing the same key again or by pressing undo. This has always worked for me. Now others may not agree with this philosophy and they may decide to implement your request and unless the implementation causes problems I won't object to that of course. But I'm just letting you know that I myself have no intention of addressing it.

**Comment by difrkaguilar** [comment]
I respect and acknowledge your judgment in adding this FR. I am even the type of user who likes to investigate the whole program thoroughly, the functions of each of the icons and associated features, in most of the 3d and graphic design programs I use I do it this way to get the best out of it. But I do not fail to recognize that from time to time there are users who post on social networks and other platforms, that they used a key combination and a panel or something disappeared. I also know that putting this clarification in the tooltips is not going to stop users like that from continuing with that behavior :-)
But it is also visually somewhat contradictory that in the same tooltip all the functions of the mouse and key combinations are clarified and especially this Shift+ctrl+n or h is not specified its functionality. Even more so when in the help it is clarified that Shift+ctrl+n or h hide the panels but in no case it is clarified that using this same combination also shows the panel that was previously hidden.

Anyway, I am very happy with the way the program is taking with each of the new features that are being integrated.

If someone else has a solid position to add or not this FR can clarify it, if not then it can be closed without any problem.

**Comment by dterrahe** [comment]
I fully agree that for the new user the distinction between mouse actions combined with modifier keys (shift/ctrl/alt), which need to be performed directly on an object, and shortcuts, that don't care where the mouse is the moment they are executed and can be configured, is not necessarily obvious. I'm not sure how much is said about this difference in the manual, but it doesn't really matter because most new users will not start out by reading the manual. I have no good solutions to offer for this.

Separately some of the actions could be named better ("toggle scope/navigator visibility"). I did not want to make too many changes when I introduced the new shortcut system, because I worried people would get lost if _everything_ had changed. But that meant an opportunity for a good cleanup was lost. Others can definitely help out here. Although it would mean that if anybody has customized those shortcuts, their changes would be lost/need to be redone.
